gradle-examples/pom.xml

```diff
@@ -11,7 +11,7 @@
   <parent>
     <groupId>io.thorntail.examples</groupId>
     <artifactId>examples-parent</artifactId>
-    <version>2.5.1.Final-SNAPSHOT</version>
+    <version>2.6.1.Final-SNAPSHOT</version>
```

Comment:
Ah, thanks :-) This didn't update because the module was commented out and I forgot about that.